cluster-autoscaler/cloudprovider/gce/gce_url.go

```diff
@@ -22,12 +22,9 @@ import (
 )
 
 const (
-	gceUrlSchema    = "https"
-	gceDomainSuffix = "googleapis.com/compute/v1/projects/"
-	// Cluster Autoscaler previously used "content" instead of "www" here, for reasons unknown.
-	gcePrefix           = gceUrlSchema + "://www." + gceDomainSuffix
-	instanceUrlTemplate = gcePrefix + "%s/zones/%s/instances/%s"
-	migUrlTemplate      = gcePrefix + "%s/zones/%s/instanceGroups/%s"
+	gceUrlSchema      = "https"
```

Comment:
Can we remain this to something like `urlPrefix` or `httpsPrefix`? The current does not accurately describe the content.

Reply:
removed it alltogether